doctordroid/README.md

```diff
@@ -0,0 +1,35 @@
+# Doctor Droid
+
+## Overview
+
+Doctor Droid is an observability analytics platform that analyzes alerts history and fetches metrics from upstream and downstream services. Use the integration to:
+- Decide better thresholds for alerts
+- Identify alerting coverage for your services
+- Enrich alerts for faster diagnosis
+
+## Setup
+
+### Installation
+Visit [Doctor Droid][1] to sign up for free. Once registered, visit the [integrations page][2]  and add a Datadog integration. This guides you through the Datadog OAUTH2 flow to grant Doctor Droid access to your API Keys to query APM & Infra metrics for you.
+
+### Configuration
+After you add the integration, explore your alerts history within Doctor Droid to discover trends. You can create reports and playbooks to enrich your generated alerts data.
+
+## Uninstallation
+
+To remove the Datadog integration from Doctor Droid:
+1. Navigate to the [Doctor Droid integrations page][2].
+1. Click  **Delete**. 
+1. Click the  **Uninstall Integration**  button to uninstall this integration from Datadog. 
```

Comment:
The user will need to do this step on the Datadog integration tile

Reply:
```suggestion
1. Navigate to the [Datadog integration tiles][4]. Find and select the integration tile for Doctor Droid. 
1. From the Doctor Droid integration tile, click the  **Uninstall Integration**  button to uninstall from Datadog. 
```